src/test/regress/sql/with.sql

```diff
@@ -1065,3 +1065,21 @@ WITH cte1 AS (
 SELECT * FROM cte1 a JOIN cte1 b USING (c1);
 
 DROP TABLE t1;
+
+-- Ensure that prefetch is not disabled for HashJoin in case of join at single segment.
+-- Test Shared Scan producer is executed under inner part of join first and the
+-- deadlock between Shared Scans does not occur
+SET optimizer = off;
+--start_ignore
+DROP TABLE IF EXISTS d;
+--end_ignore
+CREATE TABLE d (c1 int, c2 int) DISTRIBUTED BY (c1);
+
+INSERT INTO d VALUES ( 2, 0 ),( 2, 0 );
```

Comment:
```suggestion
INSERT INTO d VALUES (2, 0), (2, 0);
```

Reply:
this style is used in other tests in this file:
https://github.com/arenadata/gpdb/blob/3756bbba81d9e01ee5a8b4d2238734c47fa0c43e/src/test/regress/sql/with.sql#L979-L980